src/styles/_catalog.scss

```diff
@@ -1,21 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  gap: 40px;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
   }
 
+  @media (min-width:488px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width:768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width:1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
+
```

Comment:
Its better to use variables https://mateacademy.slack.com/archives/C03H0MLSFUJ/p1655716733610009

Reply:
I used variables, like in example)